### PR TITLE
BETA: Register shamil.is-a.dev

### PR DESCRIPTION
**Pull request by shamxl**
Added `shamil.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by Stef-DP — APPROVED** [review]
seems good now

File: domains/shamil.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "shamxl",
+           "email": "103837170+shamxl@users.noreply.github.com",
```
Comment (Stef-00012) [inline domains/shamil.json:4]: You cannot use a noreply mail in the mial field

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by Stef-DP — APPROVED** [review]
seems good now

**Review by iostpa — APPROVED** [review]
seems good to me

**Pull request merged**